src/Explorer/QueryCopilot/Modal/WelcomeModal.tsx

```diff
@@ -0,0 +1,89 @@
+import { DefaultButton, IconButton, Modal } from "@fluentui/react";
+import { useBoolean } from "@fluentui/react-hooks";
+import React from "react";
+import Database from "../../../../images/CopilotDatabase.svg";
+import Flash from "../../../../images/CopilotFlash.svg";
+import Thumb from "../../../../images/CopilotThumb.svg";
+import CoplilotWelcomeIllustration from "../../../../images/CopliotWelcomeIllustration.svg";
+import "./WelcomeModal.css";
+
+export const WelcomeModal = ({ visible }: { visible: boolean }): JSX.Element => {
+  const [isModalVisible, { setFalse: hideModal }] = useBoolean(visible);
+  const disableModal = (): void => {
+    window.localStorage.setItem("showWelcomeModal", "true");
```

Comment:
Yeah, changing to hide makes more semantical sense, since we just care if the key exists not really what it is. But with the name being hide it makes more sense.